modules/virt-configuring-vm-disk-sharing.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-disk-sharing{context}"]
+= Configuring disk sharing by using virtual machine disks
+
+You can configure a disk's block volumes to allow multiple virtual machines to share the same storage. 
+
+The application running in the guest determines the storage option you must configure for the VM. A disk of type `disk` exposes the volume as an ordinary disk to the VM.
+
+.Prerequisites 
+
+* `ReadWriteMany` (RWX) volume access mode is required if the VMs that are sharing disks are running on different nodes. If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
+
+* The storage provider must support the required Container Storage Interface (CSI) driver.
+
+.Procedure
+
+. Edit or create the `VirtualMachine` manifest for your VM to set the required values, as shown in the following example: 
```

Comment:
I think only "Edit" applies here. The user would not be able to create a VM from this YAML. We can assume that the VM already exists.

Reply:
Done